blob-clients-common/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.bazaarvoice.emodb</groupId>
         <artifactId>emodb-parent</artifactId>
-        <version>6.4.12-SNAPSHOT</version>
+        <version>${project.version}</version>
```

Comment:
this is managed by maven-release-plugin, so no need to change that

Reply:
Since the artifactId is not present in [root POM](https://github.com/bazaarvoice/emodb/blob/master/pom.xml#L59-L60) modules, the change is needed, however I'll remove it since this is a part of BlobStore module, out of scope for current ticket.  Will be adding it as part of[ PD-174028](https://bazaarvoice.atlassian.net/browse/PD-174028)